src/tests/Microsoft.PowerFx.Performance.Tests/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using BenchmarkDotNet.Running;
+
+namespace Microsoft.PowerFx.Performance.Tests
+{
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * 
+             *       --> This program needs to be run ELEVATED
```

Comment:
>--> [](http://example.com/codeflow?start=21&length=3)

does this also need to be run in Release?

Reply:
Yes, this project isn't compiled in Debug